nuclei/utils.py

```diff
@@ -48,15 +48,15 @@ def base64_to_buffer(blob: bytes) -> io.BytesIO:
     return buf
 
 
-def deserialize_polars_json(message: dict) -> Union[pd.DataFrame, pd.Series]:
+def deserialize_polars_json(message: dict) -> pl.DataFrame:
     """
     Only needed for backwards compatibility.
     """
-    return pl.read_json(message["body"])
+    return pl.read_json(io.BytesIO(message["body"].encode("utf8")))
```

Comment:
I think `io.StringIO` is faster. I assume `message["body"]` already is a `str` dtype. This guarantees we already deal with valid `utf8` so that check is redundant.

Reply:
In the new version of polars that is correct. However, `io.StringIO` gives an error in the older version of polars and I don't want to change the polars dependency in setup `"polars>=0.9.12,<0.14.0"`